packages/message-kit/src/helpers/types.ts

```
@@ -18,17 +18,19 @@
   send: (content: string, contentType?: ContentTypeId) => Promise<string>;
   isAdmin: (inboxId: string) => boolean;
   isSuperAdmin: (inboxId: string) => boolean;
+  admins: string[];
+  superAdmins: string[];
   createdAt: Date;
 };
 export type SkillResponse = {
   code: number;
   message: string;
-  data?: any;
+  data?: {} | undefined;
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)